spec/models/dish_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Dish, type: :model do
+
+  it 'should have valid Factory' do
+    expect(FactoryGirl.create(:dish)).to be_valid
+  end
+
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :alergyInfo }
```

Comment:
So, the naming standard for attributes ic snake_case :-( Sorry, I missed that in my previous review. (also, it's `allergy` with 2 `l's`)
```
allergy_info
```